spec/pardot/authentication_spec.rb

```diff
@@ -32,6 +32,45 @@ def authenticate
       authenticate
       @client.authenticated?.should == true
     end
+
+    it "should use version 3" do
+      authenticate
+      @client.version.to_i.should == 3
+    end
+    
+  end
+
+  describe "authenticateV4" do
```

Comment:
I know there's probably an easier way to do this, but my mind has been blanking on how. I've been out of rspec/ruby dev for too long.